packages/core/docs/guide/composables.md

```diff
@@ -1,200 +1,20 @@
-  }
-};
-```
-
-Let's disable a button when username or password is empty. In order to do this, we can create a computed property - and this can be also done with the Composition API inside the `setup`:
-
-```html{5,15-17}
-<template>
-  <form @submit="submitForm">
-    <input v-model="username" type="text" placeholder="Username" />
-    <input v-model="password" type="password" placeholder="Password" />
-    <button :disabled="!isValid" type="submit">Submit</button>
-  </form>
-</template>
-
-<script>
-  import { ref, computed } from 'vue';
-
-  export default {
-    setup() {
-      /*...*/
-      const isValid = computed(
-        () => username.value.length > 0 && password.value.length > 0
-      );
-
-      return { username, password, submitForm, isValid };
-    }
-  };
-</script>
-```
-
-Now, we can make a final step and abstract all the form functionality _outside_ the component - in a standalone function:
-
-```js
-// useForm.js
-
-import { ref, computed } from 'vue';
-
-export function useForm() {
-  const username = ref('');
-  const password = ref('');
-
-  const submitForm = () => {
-    console.log(username.value);
-  };
-
-  const isValid = computed(
-    () => username.value.length > 0 && password.value.length > 0
-  );
-
-  return { username, password, submitForm, isValid };
-}
-```
-
-This function is what we call a **composable**: a reusable piece of logic with the reactive state. Composables can be imported and used in Vue components:
-
-```js
-// SomeComponent.vue
-
-import { useForm } from './useForm.js';
-
-export default {
-  setup() {
-    const { username, password, submitForm, isValid } = useForm();
-
-    return { username, password, submitForm, isValid };
-  }
-};
-```
-
-Vue Storefront uses composables as its main API. We will take a look over them in the next section.
-
-## What are Vue Storefront composables
+> If you are not familiar with Composition APIs, we recommend reading [this article](/guide/composition-api) first.
 
 Composable is a function that uses [Composition API](#composition-api) under the hood. Composables are the main public API of Vue Storefront and, in most cases, the only API except configuration you will work with.
 
 You can treat composables as independent micro-applications. They manage their own state, handle server-side rendering, and rarely interact with each other (except [useUser](/composables/use-user.html) and [useCart](/composables/use-cart.html)). No matter what integration you are using, your application will always have the same set of composables with the same interfaces.
 
-To use a composable, you need to import it from an integration you are using, and call it on the component `setup` option:
+To use a composable, you need to import it from an integration you use, and call it on the component `setup` option:
 
 ```js
-import { useProduct } from '{INTEGRATION}';
+import { useProduct } from '@vue-storefront/commercetools'; // or any other integration package
```

Comment:
I suggest sticking with `{INTEGRATION}` to be consistent with the rest of the pages. `@vue-storefront/commercetools` might be harder to catch for people using different integrations.